api/src/main/java/org/openmrs/module/kenyaemr/reporting/cohort/definition/CACXRegisterCohortDefinition.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.kenyaemr.reporting.cohort.definition;
+
+import org.openmrs.Encounter;
+import org.openmrs.module.reporting.cohort.definition.BaseCohortDefinition;
+import org.openmrs.module.reporting.common.Localized;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationProperty;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationPropertyCachingStrategy;
+import org.openmrs.module.reporting.evaluation.caching.Caching;
+import org.openmrs.module.reporting.query.BaseQuery;
+import org.openmrs.module.reporting.query.encounter.definition.EncounterQuery;
+
+import java.util.Date;
+
+/**
+ * Cervical cancer cohort definition
+ */
+@Caching(strategy = ConfigurationPropertyCachingStrategy.class)
+@Localized("reporting.CACXRegisterCohortDefinition")
+public class CACXRegisterCohortDefinition extends BaseQuery<Encounter> implements EncounterQuery {
+    @ConfigurationProperty
+    private Date startDate;
```

Comment:
we don't need these here. Remove even the getters and setters. We are already passing the start and end dates using params